Manifest.toml

```diff
@@ -0,0 +1,64 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Any particular reason you want to check in the Manifest file? I thought best practice for projects was to only check in the Project file.

Reply:
Checking in the `Manifest.toml` gives insurance against someone who just wants to get things working in the future being surprised when there are no upper bounds and things don't work.  They can just `instantiate` and get the exact versions that a package had the last time someone dev'ed on it.  It's kind of like carrying around a "known good point", even if there are thousands of possible good points in the package space, here's at least one that works.